Userland/Libraries/LibWeb/Painting/PaintableBox.cpp

```diff
@@ -349,14 +369,11 @@ void PaintableBox::before_children_paint(PaintContext& context, PaintPhase phase
     }
 }
 
-void PaintableBox::after_children_paint(PaintContext& context, PaintPhase phase, ShouldClipOverflow should_clip_overflow) const
+void PaintableBox::after_children_paint(PaintContext& context, PaintPhase phase) const
 {
     if (!AK::first_is_one_of(phase, PaintPhase::Background, PaintPhase::Border, PaintPhase::Foreground))
         return;
 
-    if (should_clip_overflow == ShouldClipOverflow::No)
-        return;
-
```

Comment:
What's the reasoning for removing `should_clip_overflow`? (Here and elsewhere)

Reply:
seems like it was added to always prevent clipping for absolutely positioned elements but actually absolutely positioned element can be clipped if it's containing block has `overflow: hidden` and it's work fine when clip rectangle is computed not from parent but from containing block like I made it work in this patch (I updated commit message with a note about `should_clip_overflow`)